tests/break-calls/jsfmt.spec.js

```diff
@@ -1 +1,2 @@
 run_spec(__dirname);
+run_spec(__dirname, {parser: 'typescript'});
```

Comment:
Would you be open to an initial PR to add tooling around enforcing that? Is there a particular reason why linting or an .editorconfig has not be added up to this point?

I could also improve the consistency in file and folder naming. Right now snake, dash and camel case are all used which made putting this PR together a little more difficult.